paymentsheet/src/main/java/com/stripe/android/paymentsheet/analytics/EventReporter.kt

```diff
@@ -176,6 +176,18 @@ internal interface EventReporter {
         error: Throwable,
     )
 
+    /**
+     * The customer has successfully set a payment method as the default
```

Comment:
```suggestion
     * The customer has successfully set a payment method as the default.
```

nit: for consistency with the other javadoc in this file